NAMESPACE

```diff
@@ -9,10 +9,12 @@ export(create_glossed_document)
 export(create_presentation)
 export(create_subannotation)
 export(create_viewer)
+export(crocodile)
```

Comment:
I'm afraid this function should be deleted. I'd rather remove this fork and start a new one from the scratch: this will remove our learning step from the package history.